### ISSUE TITLE
Query REST Function

### ISSUE THREAD

**Issue by ericderegt** [NYU-DevOps-Spring2018-Orders/orders#9]
**As a** user  
**I need** to access a query endpoint   
**So that** I can find orders  

**Assumptions:**
* /orders/<int:id> is the route

**Acceptance Criteria:**
```
When I run the service locally and request /orders/id,
an individual order should be returned.

Rest function should also be set up with a test case to ensure function is working properly
```

**Comment by hjcho7311** [comment]
Duplicate of Issue #24

**Comment by hjcho7311** [comment]
reworking for this issue... closed #24
